### DISCUSSION TITLE
Use of Apache Hive in luarocks

### DISCUSSION THREAD

**Discussion by kollol-chowdhury-incontact** [keplerproject/luarocks#1403] [Q&A]
Hi Team,

Want to know the use of Apache Hive in Luarocks as we are using Kong and its detecting vulenrability in Apache Hive 
argparse
binaryheap
lua-resty-http
lua-resty-jit-uuid
lua-resty-timer
lua-resty-worker-events
luasocket
luaxxhash

Please help.

TIA

**Comment by kollol-chowdhury-incontact** [comment]
CVE-2021-45232
CVE-2022-24112
CVE-2021-43557

These are the vulnerabilities reported.

**Comment by hishamhm** [comment]
@kollol-chowdhury-incontact LuaRocks does not use Apache Hive, and all three CVE's you linked above refer to Apache APISIX, not LuaRocks (or Kong, for that matter).
